doc/user/content/self-managed/installation/install-on-aws.md

```diff
@@ -14,101 +19,237 @@ For testing purposes only. For testing purposes only. For testing purposes only.
 
 ## Prerequisites
 
+### Terraform
+
+If you don't have Terraform installed, [install
+Terraform](https://developer.hashicorp.com/terraform/install?product_intent=terraform).
+
+### AWS CLI
+
+If you do not have the AWS CLI installed,
+
+- Install the AWS CLI. For details, see the [AWS
+  documentation](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html).
+
+- Configure with your AWS credentials. For details, see the [AWS
+  documentation](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html).
+
+### kubectl
+
+If you do not have `kubectl`, install. See the [Amazon EKS: install `kubectl`
+documentation](https://docs.aws.amazon.com/eks/latest/userguide/install-kubectl.html)
+for details.
+
+### Helm 3.2.0+
+
+If you do not have Helm 3.2.0+, install. See the [Helm
+documentation](https://helm.sh/docs/intro/install/).
+
 ### AWS Kubernetes environment
 
+{{% self-managed/requirements-list %}}
+
 When operating in AWS, we recommend:
 
+- Using the `r8g`, `r7g`, and `r6g` families when running without local disk.
+
 - Using the `r7gd` and `r6gd` families of instances (and `r8gd` once available)
-  when running with local disk
+  when running with local disk (Recommended for production.  See [Operational guidelines](/self-managed/operational-guidelines/#locally-attached-nvme-storage-openebs) for more information.)
+
+See [A. Set up AWS Kubernetes environment](#a-set-up-aws-kubernetes-environment)
+for a sample setup.
 
-- Using the `r8g`, `r7g`, and `r6g` families when running without local disk
+## A. Set up AWS Kubernetes environment
 
-[//]: # "TODO: Add Terraform and non-Terraform instructions here (tabbed)."
+{{< tabs  >}}
 
-Materialize provides a [sampleTerraform
+{{< tab "Terraform" >}}
+
+Materialize provides a [sample Terraform
 module](https://github.com/MaterializeInc/terraform-aws-materialize/blob/main/README.md)
-to deploy a sample infrastructure on AWS with the following components:
+for evaluation purposes only. The module deploys a sample infrastructure on AWS
+(region `us-east-1`) with the following components:
 
 - EKS component
 - Networking component
```

Comment:
not sure what "networking component" translates to. I think we can drop "component" from each of these lines:

```
- A dedicated VPC
- A Kubernetes (EKS) cluster
- An S3 bucket
- An RDS Postgres cluster and database
```

Reply:
Updated in patch